src/pages/SSLModule/create/components/CertificateUploader/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Form, Button, Upload } from 'antd';
+import { UploadOutlined } from '@ant-design/icons';
+import { UploadChangeParam } from 'antd/lib/upload';
+import { UploadFile } from 'antd/lib/upload/interface';
+import { useForm } from 'antd/es/form/util';
+import forge from 'node-forge';
+import { FormData } from '../..';
+import styles from '../../style.less';
+
+interface AltName {
+  value: string;
+}
+
+export type UploadType = 'PUBLIC_KEY' | 'PRIVATE_KEY';
+export interface SuccessDataProps {
+  sni?: string;
+  cert?: string;
+  key?: string;
+  expireTime?: Date;
+  publicKeyDefaultFileList?:
+    | []
+    | [
+        {
+          uid: string;
+          status: string;
+          name: string;
+        },
+      ];
+  privateKeyDefaultFileList?:
+    | []
+    | [
+        {
+          uid: string;
+          status: string;
+          name: string;
+        },
+      ];
+}
+
+interface UploaderProps {
+  onSuccess(uploadData: SuccessDataProps): void;
+  onDelFile(type: UploadType): void;
+  data: FormData;
+}
+
+const CertificateUploader: React.FC<UploaderProps> = ({ onSuccess, onDelFile, data }) => {
+  const [form] = useForm();
+  const handleChange = (info: UploadChangeParam<UploadFile<any>>, type: UploadType) => {
+    if (!info.file.originFileObj) return;
+    const fileRead = new FileReader();
+    fileRead.readAsText(info.file.originFileObj);
+    // eslint-disable-next-line func-names
+    fileRead.onload = function (event) {
+      const fileResult = (event.currentTarget as any).result;
```

Comment:
result

Reply:
> result

![image](https://user-images.githubusercontent.com/31329157/82125088-1d625300-97d6-11ea-875e-cc787ef9ae57.png)

It can't be 'result' here.

Reply:
....

```ts
const {result} = event.xxx as any
```